suite-native/notifications/src/components/NotificationRenderer.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { ReactNode } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { useSelector } from 'react-redux';
+
+import { Box, VStack } from '@suite-native/atoms';
+import {
+    NotificationsRootState,
+    selectTransactionNotifications,
+} from '@suite-common/toast-notifications';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { TransactionNotification } from './TransactionNotification';
+
+type NotificationRendererProps = {
+    children: ReactNode;
+};
+
+const notificationContainerStyle = prepareNativeStyle<{ topSafeAreaInset: number }>(
+    (utils, { topSafeAreaInset }) => ({
+        position: 'absolute',
+        flex: 1,
+        flexDirection: 'row',
+        marginTop: topSafeAreaInset,
+        paddingHorizontal: utils.spacings.medium,
+    }),
+);
+
+export const NotificationRenderer = ({ children }: NotificationRendererProps) => {
+    const { applyStyle } = useNativeStyles();
+    const { top: topSafeAreaInset } = useSafeAreaInsets();
+    const transactionNotifications = useSelector((state: NotificationsRootState) =>
```

Comment:
if you don't have extra params you should be able to do just `useSelector(selectTransactionNotifications)`

Reply:
fixed 53e695678